docs(tracing): Add missing searchable span properties (#10879)
getsentry/sentry-docs: docs/concepts/search/searchable-properties/spans.mdx

### `action`

> The type of span action, e.g `SELECT` for a SQL span or `POST` for an HTTP span.

Reviewer: ```suggestion
The type of span action, for example, `SELECT` for a SQL span or `POST` for an HTTP span.
```

- **Type:** string

### `domain`

> General scope of the span’s action, i.e. the tables involved in a `db` span or the host name in an `http` span.

Reviewer: ```suggestion
General scope of the span’s action, for example, the tables involved in a `db` span or the host name in an `http` span.
```

- **Type:** string

### `module`

> The Insights module that the span is associated with, e.g `cache`, `db`, `http`, etc.

Reviewer: ```suggestion
The Insights module that the span is associated with, for example, `cache`, `db`, `http`, and so on.
```

- **Type:** string

### `op`

> The operation of the span, e.g `http.client`, `middleware`.

Reviewer: ```suggestion
The operation of the span, for example, `http.client`, `middleware`.
```
